PassthruAndBypass/main.cpp

```diff
@@ -0,0 +1,78 @@
+/** Daisy Seed2 DFM Pedal Dev Kit Basic Example
+ *
+ *  This example does a few things that can serve as a great jumping off
+ *  point for new effects pedal projects.
+ *
+ *  1. Initializes Switch and GPIO for control of the true-bypass relays
+ *  2. Initializes an LED to indicate bypass state (illuminate RED when audio runs through software)
+ *  3. Sets up basic audio passthru
+ */
+#include "daisy_seed.h"
+
+using namespace daisy;
+
+/** Global objects that need to be accessed in both main() and the audio callback */
+DaisySeed hardware;
+
+/** The audio callback that fires whenever new audio samples can be prepared */
+void AudioCallback(AudioHandle::InputBuffer in,
+                   AudioHandle::OutputBuffer out,
+                   size_t size)
+{
+    for (size_t i = 0; i < size; i++)
+    {
+        /** For each sample in the loop we'll multiply the input by our volume control
+         *  OUT_x and IN_x are macros that access the out, and in buffers respectively.
+         *  This would be equivalent to: out[0][i] = in[0][i] * volume;, etc.
+         */
+        OUT_L[i] = IN_L[i];
+        OUT_R[i] = IN_R[i];
+
+        /** Due to minor errata on the Rev4 hardware we need to phase invert the
+         *  left channel so both outputs have the same phase */
+        OUT_L[i] *= -1.f;
+    }
+}
+
+int main()
+{
+
+    /** Initialize the Daisy Seed at maximum clock speed */
+    hardware.Init(true);
+
+    /** Relay Control/UI init */
+    GPIO relay_left, relay_right;
+    Switch relay_control;
```

Comment:
Do we want to use the two-way toggle or the foot switch for this? I initially went with the toggle because it has the LED directly above it.
The toggle's a little "easier" to operate, but the footswitch is a more idiomatic control for this use.